css/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  margin-top: 4rem;
+  width: 100vw;
+  color: white;
+  background-color: blueviolet;
+  font-family: sans-serif;
+}
+
+header {
+  margin: 1rem auto;
+  text-align: center;
+}
+
+#title {
+  font-size: 2rem;
+  text-transform: uppercase;
+}
+
+#survey-form {
+  width: 80vw;
+  padding: 2rem;
+  margin: 1rem auto;
+  border-radius: 0.5rem;
+  background-color: rgba(106, 0, 106, 0.807);
+}
+
+/* label {
+  padding-left: 0.1rem;
+  margin: 1rem 0;
+} */
```

Comment:
Avoid making unused CSS code as a comment. Using unused CSS code as a comment renders your project code unclean.

Reply:
> Avoid making unused CSS code as a comment. Using unused CSS code as a comment renders your project code unclean.

- [ ] Excellent work in requesting this change 👏. You used the inline comment correctly, and you explained why you requested the change in your comment. However, I would suggest that you edit this comment to be more polite and general, rather than limiting it to CSS, because the student may believe that it is only bad CSS practice. As a result, you can edit your comment as follows: 👇️
```
In my opinion, it would be better to remove this commented code because leaving commented code in your code base is not a good practice and makes your code look cluttered. 😊
```

- [ ] Please add another inline comment to have at least two comments as requested, and make sure it is polite as well as explaining `why` you requested the change, as you did in the previous comment. 👌😊

- [ OPTIONAL ] I would recomment to add some emojis to your comments to make them look more freindly.👍️